src/Symfony/Component/Mailer/Tests/Exception/UnsupportedSchemeExceptionTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Tests\Exception;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\PhpUnit\ClassExistsMock;
+use Symfony\Component\Mailer\Bridge\Amazon\Transport\SesTransportFactory;
+use Symfony\Component\Mailer\Bridge\Google\Transport\GmailTransportFactory;
+use Symfony\Component\Mailer\Bridge\Mailchimp\Transport\MandrillTransportFactory;
+use Symfony\Component\Mailer\Bridge\Mailgun\Transport\MailgunTransportFactory;
+use Symfony\Component\Mailer\Bridge\Mailjet\Transport\MailjetTransportFactory;
+use Symfony\Component\Mailer\Bridge\Postmark\Transport\PostmarkTransportFactory;
+use Symfony\Component\Mailer\Bridge\Sendgrid\Transport\SendgridTransportFactory;
+use Symfony\Component\Mailer\Bridge\Sendinblue\Transport\SendinblueTransportFactory;
+use Symfony\Component\Mailer\Exception\UnsupportedSchemeException;
+use Symfony\Component\Mailer\Transport\Dsn;
+
+final class UnsupportedSchemeExceptionTest extends TestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        ClassExistsMock::register(__CLASS__);
+    }
+
+    protected function setUp(): void
+    {
+        ClassExistsMock::withMockedClasses([
+            GmailTransportFactory::class => false,
+            MailgunTransportFactory::class => false,
+            MailjetTransportFactory::class => false,
+            MandrillTransportFactory::class => false,
+            PostmarkTransportFactory::class => false,
+            SendgridTransportFactory::class => false,
+            SendinblueTransportFactory::class => false,
+            SesTransportFactory::class => false,
+        ]);
+    }
```

Comment:
It looks like this is not working, but only for mailer 🤔 
Is this not available for 4.4?

cc @derrabus